cmd/luna/app/import.go

```diff
@@ -9,8 +9,32 @@ import (
 	"github.com/quay/claircore/libvuln"
 	"github.com/quay/zlog"
 	"github.com/rs/zerolog"
+	"github.com/spf13/cobra"
 )
 
+func NewImportCommand() *cobra.Command {
```

Comment:
⚠️ **[sonarqube]** <[external_golint:Exported](http://localhost:19000/project/issues?id=luna-code-enhancement&issues=AXgQ5_GMvT_5avwPlAsB)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>exported function NewImportCommand should have comment or be unexported